wiki/Configuration:-Overview.md

```diff
@@ -25,6 +25,8 @@ You can run `niri validate` to parse the config and see any errors.
 
 To use a different config file path, pass it in the `--config` or `-c` argument to `niri`.
 
+You can also set `$NIRI_CONFIG` to the path of the config file. `--config` always takes precedence. If `--config` doesn't point to a real file, the config will not be loaded. If `$NIRI_CONFIG` doesn't point to a real file, it is ignored and the default path is used instead.
```

Comment:
Why is `$NIRI_CONFIG` ignored if missing, instead of acting the same way as `--config`?

Reply:
My main thought here is that a user can wish to remove this on an individual invocation, which is slightly more annoying if you need to unset the environment variable as opposed to something like `NIRI_CONFIG="" niri` which sets it to an empty string; i.e. an invalid path. It could act the same as `--config`, it's just not what i felt would be the easiest for me to use, personally. Something like `env -u NIRI_CONFIG niri` should be an actual equivalent way to run it by unsetting `NIRI_CONFIG`, the only downside to this being that i had to actually look at documentation to find what magic incantation unsets an environment variable (whereas the env prefix is standard and common in all shells). That's a very artificial reason to do it this way, but it is my thought process.

Reply:
Hm, that is a valid point.

Reply:
But also, it can explicitly check for is_empty(), rather than path existing?

Reply:
Did that